Backends/PDF/paper.lisp

```diff
@@ -1,33 +1,31 @@
-
 (in-package #:clim-pdf)
 
 (defparameter *paper-sizes*
-  '((:letter 612 . 792)
-    (:legal 612 . 1008)
-    (:a0 2380 . 3368)
-    (:a1 1684 . 2380)
-    (:a2 1190 . 1684)
-    (:a3 842 . 1190)
-    (:a4 595 . 842)
-    (:a5 421 . 595)
-    (:a6 297 . 421)
-    (:a7 210 . 297)
-    (:a8 148 . 210)
-    (:a9 105 . 148)
-    (:a10 74 . 105)
-    (:b0 2836 . 4008)
-    (:b1 2004 . 2836)
-    (:b2 1418 . 2004)
-    (:b3 1002 . 1418)
-    (:b4 709 . 1002)
-    (:b5 501 . 709)
-    (:11x17 792 . 1224)))
+  '((:letter . (612  . 792))
+    (:legal  . (612  . 1008))
+    (:a0     . (2380 . 3368))
+    (:a1     . (1684 . 2380))
+    (:a2     . (1190 . 1684))
+    (:a3     . (842  . 1190))
+    (:a4     . (595  . 842))
+    (:a5     . (421  . 595))
+    (:a6     . (297  . 421))
+    (:a7     . (210  . 297))
+    (:a8     . (148  . 210))
+    (:a9     . (105  . 148))
+    (:a10    . (74   . 105))
+    (:b0     . (2836 . 4008))
+    (:b1     . (2004 . 2836))
+    (:b2     . (1418 . 2004))
+    (:b3     . (1002 . 1418))
+    (:b4     . (709  . 1002))
+    (:b5     . (501  . 709))
+    (:11x17  . (792  . 1224))))
```

Comment:
given you want to align columns, please align also numbers, like

```
(12345 .   42)
(   22 .  142)
```

Reply:
touché. done.